README.md

```diff
@@ -266,17 +334,23 @@ kubectl logs -f $(kubectl get pods -o=jsonpath='{.items[0].metadata.name}' \
 Let's post a session for `Service 3` into CAS.
 
 ```bash
+# Provide CLI address for next commands
+export SCONE_CLI_ADDRESS=$(kubectl -n scone get pods \
+  -o=jsonpath='{.items[0].metadata.name}' -l app=scone-cli)
+
+# Copy over Service 3 session
 kubectl cp services/3_secret_service/network-shielding-session.yaml \
-    scone/$(kubectl get pods -n scone -o=jsonpath='{.items[0].metadata.name}' \
-    -l app=scone-cli):3_network-shielding-session.yaml
+    scone/$SCONE_CLI_ADDRESS:3_network-shielding-session.yaml
 
-kubectl exec -it -n scone $(kubectl get pods -n scone -o=jsonpath='{.items[0].metadata.name}' \
-    -l app=scone-cli)  -- /bin/bash
+# SSH into cli container
+kubectl exec -it -n scone $SCONE_CLI_ADDRESS -- /bin/bash
 
 # Inside the scone-cli container
 HASH_SERVICE_3=$(scone session create 3_network-shielding-session.yaml --use-env)
-# Get the session hash for Service 2
-echo $HASH_SERVICE_3
+
+# Get the session hash for Service 3
+echo "HASH_SERVICE_3=$HASH_SERVICE_3"
```

Comment:
I think we can have a quick win here.

`echo "export HASH_SERVICE_3=$HASH_SERVICE_3"`

then ask the user to copy the entire line. It is a small change that gains the user's sympathy. The same can be done for service 2.